web-client/src/views/FileHandlingHelpers/pdfValidation.ts

```diff
@@ -29,14 +51,29 @@ export const validatePdf = ({
       const { result } = fileReader;
 
       if (!result || typeof result === 'string') {
-        resolve({
-          errorInformation: {
-            errorMessageToDisplay: GENERIC_FILE_ERROR_MESSAGE,
-            errorMessageToLog: `${GENERIC_FILE_ERROR_MESSAGE} (Failed to read file as ArrayBuffer.)`,
-            errorType: ErrorTypes.UNKNOWN,
-          },
-          isValid: false,
-        });
+        if (!clientSupportsES2022()) {
```

Comment:
If it turns out we have to put `clientSupportsES2022` above, is putting here too redundant?